lib/web_ui/lib/src/engine/window.dart

```diff
@@ -339,25 +342,6 @@ class EngineSingletonFlutterWindow extends EngineFlutterWindow {
   double? _debugDevicePixelRatio;
 }
 
-/// A type of [FlutterView] that can be hosted inside of a [FlutterWindow].
-class EngineFlutterWindowView extends ui.FlutterWindow {
```

Comment:
I like this simplification, however we left this here because it seemed that this was going to be used by the new multi-view hierarchy of classes. What happened? Is there a design doc for this?

Reply:
It isn't being used right now. If it's needed it'll be easy to re-implement.

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#lazy-programming